test/integration/serverless/run.sh

```diff
@@ -138,6 +153,7 @@ trace_functions=(
     "trace-python"
     "trace-java"
     "trace-go"
+    # traces are not supported for .net lambda function on ARM64 architecture (snapshot is empty)
```

Comment:
Since we are now skipping C# tests on ARM, I don't think you need this comment any more.

Reply:
fixed